.github/workflows/azure-vm.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy to an Azure Virtual Machine
+
+env:
+  OUTPUT_PATH: ${{ github.workspace }}
+  DOTNET_VERSION: '6.0.x'
+
+
+on: push
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up .NET Core
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+
+      - name: Publish with dotnet
+        run: dotnet publish -c Release -r linux-x64 --self-contained true
+        working-directory: HTTPServerProject
+
+      - name: Run a multi-line script
```

Comment:
The name should be what this multi-line script does. Looks like it tars up the build.

Reply:
Agreed.